app/src/main/java/org/simple/clinic/di/NetworkModule.kt

```diff
@@ -104,8 +104,10 @@ open class NetworkModule {
   @AppScope
   fun retrofit(commonRetrofitBuilder: Retrofit.Builder): Retrofit {
     val baseUrl = BuildConfig.API_ENDPOINT
+    val currentApiVersion = "v3"
+
     return commonRetrofitBuilder
-        .baseUrl(baseUrl)
+        .baseUrl("$baseUrl$currentApiVersion/")
```

Comment:
We needed to add this "/" at the end here because Retrofit complains if the base url does not end with a forward slash.